src/app/components/SearchHeader/components/SearchButton/styles.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  headerRight: {
+    maxHeight: 30,
+    maxWidth: 30,
```

Comment:
extract common constant value

Reply:
Done! (I don't like me that approach but, i changed it anyway)